resources/api-gateway-migrator/templates/job.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "api-gateway-migrator.name" . }}-job
+spec:
+  backoffLimit: 1
+  template:
+    metadata:
+      name: {{ template "api-gateway-migrator.name" . }}-job
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: {{ template "api-gateway-migrator.name" . }}-account
+      restartPolicy: Never
+      containers:
+        - name: migrator
+          image: colunira/api-migrator:cmd
```

Comment:
Please use the image built by CI. The component is now plugged into prow.

Reply:
Done.